operations/configuration_management/iSCSI_SBPS_Node_Personalization.md

```diff
@@ -150,6 +150,14 @@ The LUN projection is over either HSN or NMN.
     }
     ```
 
+Note:
+For specific target worker nodes selection for node personalization (instead of default: to all workers)
+use CFS option `--ansible-limit` with xnames of the worker nodes while creating a session.
+
+```bash
+cray cfs v3 sessions create --name <session name> --configuration-name <config name> --ansible-limit <xname1,xname2,...>
```

Comment:
Please change the space to an underscore, so it is `<session_name>` and `<config_name>`, which makes it clearer that it is a single argument.